src/Algorithms/FLOWSTAR/post.jl

```diff
@@ -0,0 +1,86 @@
+using .Flowstar: FlowstarSetting, ContinuousReachModel, FlowstarContinuousSolution,
+                 AbstractTMOrder, FixedTMOrder, AbstractPreconditioner, QRPreconditioner,
+                 AbstractODEScheme, NonPolyODEScheme
+
+function FLOWSTAR(; δ::Union{Float64,Nothing}=nothing,
+                    step_size::Union{Float64, IA.Interval{Float64},Nothing}=nothing,
+                    order::AbstractTMOrder=FixedTMOrder(6),
+                    remainder_estimation=1e-4,
+                    precondition::AbstractPreconditioner=QRPreconditioner(),
+                    cutoff::Float64=1e-20,
+                    precision=53,
+                    verbose=false,
+                    scheme=NonPolyODEScheme())
+    @requires Flowstar
+
+    step_size = !isnothing(δ) ? δ : (!isnothing(step_size) ? step_size : throw(ArgumentError("the step size should be specified")))
+    FLOWSTAR{typeof(step_size), typeof(order), typeof(precondition), typeof(scheme)}(
+        step_size, order, remainder_estimation, precondition, cutoff, precision, verbose, scheme)
+end
+
+function post(alg::FLOWSTAR{ST,OT,PT, IT}, ivp::IVP{<:AbstractContinuousSystem}, timespan;
+              Δt0::TimeInterval=zeroI, kwargs...) where {ST,OT,PT,IT}
+    @requires Flowstar
+
+    @unpack step_size, order, remainder_estimation, precondition, cutoff, precision, verbose, scheme = alg
+
+    # initial time and final time
+    t0, T = tstart(timespan), tend(timespan)
+    @assert t0 == 0.0
+
+    # extract model file
+    model = if isa(ivp.s, BlackBoxContinuousSystem{String})
+        # the model is provided in .flow form
+        # consistency of t0, T, X0 with the model is not checked
+        ivp.s.f
+    else
+        s, n = system(ivp), statedim(ivp)
+
+        # vector field in string form
+        @requires Symbolics
+        fstr = convert(Vector{Tuple{String,String}}, s)
+        
+        # initial set
+        X0 = initial_state(ivp)
+        dom = isa(X0, IntervalBox) ? X0 : convert(IntervalBox, overapproximate(X0, Hyperrectangle))
+
+        name = "flowstar" # TODO parse name of f
+
+        # Plot first two variables if possible, otherwise plot the first variable vs time
+        # TODO disable this option
+        plot_states = n > 1 ? "$(fstr[1][1]),$(fstr[2][1])" : "t,$(fstr[1][1])" # seems mandatory?
+        gen_plots = false # ignored?
```

Comment:
FYI, in conversation with Chen he mentioned that to turn plotting off you need to prepend the `output` setting as `tm ouput` *however* a valid plot setting is still required at parse time. That is why `plot_states` is requred.

Reply:
Ok, thanks for the clarification.